src/ui/components/ReactReduxPanels.module.css

```diff
@@ -0,0 +1,42 @@
+.Body {
+  flex: 1 1 auto;
+}
+
+.ResizeHandle {
+  height: 0.75rem;
+}
+.ResizeHandleBar {
+  width: 100%;
+  height: 1px;
+  background-color: var(--theme-border);
+}
+
+.DispatchDetailsPanel {
+  min-height: 1.75rem;
+}
+
+.DispatchDetailsHeader {
+  font-size: var(--font-size-large);
+  display: flex;
+  align-items: center;
+  gap: 1ch;
+  cursor: pointer;
+  padding: 0 0.75rem;
+}
+
+.DispatchDetailsContainer {
+  height: 100%;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  overflow: auto;
+  gap: 0.75rem;
+}
+
+.ListContainer {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  padding: 0;
+  overflow: auto;
+}
```

Comment:
Can we delete this file? Seems like nothing is referencing it.

Reply:
It gets used in the follow-on PR - got accidentally included here as I was trying to split up the code.

Reply:
Kill it from this PR?